hack/aks/Makefile

```diff
@@ -212,6 +217,71 @@ swift-up: rg-up swift-net-up ## Bring up a SWIFT AzCNI cluster
 		--yes
 	@$(MAKE) set-kubeconf
 
+vnetscale-swift-byocni-up: rg-up vnetscale-swift-net-up ## Bring up a Vnet Scale SWIFT BYO CNI cluster
```

Comment:
We might want a warning or something that this isn't enabled automatically except for specific environments.

Reply:
Added a comment to reference the requirements to enable this feature and create Vnet Scale clusters